ParquetSharp.DataFrame/DataFrameExtensions.cs

```diff
@@ -8,58 +8,100 @@ namespace ParquetSharp
     public static class DataFrameExtensions
     {
         /// <summary>
-        /// Write a DataFrame in Parquet format
+        /// Writes a Dataframe in Parquet format using existing writer.
```

Comment:
TODO: add unit tests.

Reply:
:+1: could you also please document this feature in the README file